sources/europe/hr/gis-rijeka-2000.geojson

```diff
@@ -0,0 +1,114 @@
+{
+    "type": "Feature",
+    "properties": {
+        "category": "historicphoto",
+        "id": "gis-rijeka-2000",
+        "name": "gis-rijeka: Rijeka 2000 aerial imagery",
+        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/hr/rijeka.png",
+        "start_date": "2000",
+        "end_date": "2000",
+        "url": "http://gis01.rijeka.hr/geoserver/CKAN/wms?service=WMS&version=1.1.0&request=GetMap&layers=Ortofoto_2000&styles=&bbox={bbox}&width={width}&height={height}&srs={proj}&format=image/png",
+        "permission_osm": "explicit",
+        "license_url": "http://opendefinition.org/licenses/cc-by/",
```

Comment:
The `license_url` should point to an URL that describes under what license the data can be used, not the license itself.